### DISCUSSION TITLE
Feat req: Television

### DISCUSSION THREAD

**Discussion by fantismo** [leepeuker/movary#580] [Ideas]
This is a really nice web app and my hat is off to you. After testdriving the demo, I'm definitely going to add this to my personal server. Thanks very much for making this accessible. It's greatly appreciated.

It'd be nice if it also did television. But I'll be running it either way.

Cheers!

**Comment by leepeuker** [comment]
Hey @fantismo, thanks for the nice words :) 
There is an issue about the topic TV shows here: https://github.com/leepeuker/movary/issues/186

Honestly I do not think that this feature will come. There are so many basic things still missing or only half finished and I have a lot more ideas I want to add later without having to worry about implementing and maintaining TV show support (which would complicate things much more). I have a full time job and a family, so there is only a pretty limited amount of time I can invest into Movary.

**Comment by fantismo** [comment]
Makes sense. Sounds good. Best wishes!

**Comment by Drallas** [comment]
+1 for Tv support for what it’s worth!
